.github/workflows/php.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         run: composer install --ansi --prefer-dist --no-interaction --no-progress
 
       - name: Run phpunit (without DB)
-        run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml
+        run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml; git diff --exit-code
```

Comment:
Why put this in the same githubaction step? Should'nt we make it a specific step?

Reply:
As it is phpunit that remove some files I thought it is a part of this Action to ensure the state is the same at the beginning and at the end.

What do you recommend ? 
Should I run two distinct Actions with 
```yaml
         run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml
```
and
```yaml
         run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml; git diff --exit-code
```
?

Or maybe we can assume that an Action is dependent of preceding Action ? :thinking: